keanu-python/tests/test_model.py

```diff
@@ -20,6 +20,10 @@ def test_to_bayes_net() -> None:
     assert m.sigma.get_id() in net_vertex_ids
     assert m.gaussian.get_id() in net_vertex_ids
 
+    assert m.mu.get_label().getQualifiedName() == "mu"
```

Comment:
here and elsewhere in the tests - you should use snake case `get_qualified_name()`

Reply:
can't do that yet, `get_label()` returns a raw Java Object still cause your `VertexLabel` stuff isn't merged yet